src/usr/local/buildpack/tools/v2/node.sh

```diff
@@ -41,10 +41,13 @@ function install_tool () {
   local npm # temp npm executable
   local arch=linux-x64
 
-  checksums=$(get_from_url "https://nodejs.org/dist/v${TOOL_VERSION}/SHASUMS256.txt")
+  local checksum_file
+  local original_checksum
+
+  checksum_file=$(get_from_url "https://nodejs.org/dist/v${TOOL_VERSION}/SHASUMS256.txt")
```

Comment:
didn't we merged that already?

Reply:
And I am not up to date, let me rebase...

Reply:
I am... disregard my last comment.
As I said, I changed the naming to be consistent

Reply:
please let's do that in separate refactor PR